collections/README.adoc

```diff
@@ -1,3 +1,51 @@
 = Collections good practices
 
-NOTE: Work in Progress...
+Note: Unreviewed work. Please contribute to the discussion in the Automation Red Hat COP
+
+=== Collection Structure
+[%collapsible]
+====
+Explanations:: Collections should be comprised of roles collected either at the type or landscape level. See https://github.com/redhat-cop/automation-good-practices/blob/main/structures/README.adoc[the structures definition]
+
+Rationale:: Gathering and publishing collections, rather than individual roles, allows for easier distribution and particularly becomes more important when we discuss Execution Environments.
+
+====
+=== Importing Collections
+[%collapsible]
+====
+Explanations:: Collections should always be imported using requirements files within the role
```

Comment:
After having read the linked article, I'm still not sure I understand which part of it you want to highlight, and from what I've read I'm not sure I agree with the statement.
I see two main statements in the blog article:
1. refer to collections dependencies (and roles) by name _and_ version
2. make sure to install roles and collections relative to your playbook/project/Git repo and _not_ in a central place

My issue and it's also due to a lack of experience so far, is how well those recommendations would work in the context of execution environments/controller/navigator:
1. how does controller/navigator react to collections not being installed where expected (or would the location be anyway overwritten)
2. how to handle conflicting versions between EE and requirements.yml?
3. an EE comes with a set version of some collections, so would your requirements.yml, you might need to change the version in two places
4. does navigator/controller recognize at all that a collection requested in requirements.yml is already available in the EE?

So, in short, we need to discuss all these aspects before we can take a decision.

Reply:
Agreed, removing for now